checker/src/main/java/alluxio/checker/SparkIntegrationChecker.java

```diff
@@ -0,0 +1,320 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.checker;
+
+import alluxio.PropertyKey;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.SparkConf;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import scala.Serializable;
+import scala.Tuple2;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import java.text.SimpleDateFormat;
+
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * The Spark integration checker includes a spark job to test the integration of Spark with Alluxio.
+ * This class will be triggered in client and cluster mode through spark-submit.
+ *
+ * This checker requires a running Spark cluster, but does not require a running Alluxio cluster.
+ * It will check whether Alluxio classes and Alluxio filesystem can be recognized in Spark
+ * driver and executors.
+ */
+public class SparkIntegrationChecker implements Serializable{
+  private static final long serialVersionUID = 1106074873546987859L;
+
+  @Parameter(names = {"--partitions"}, description = "The user defined partition number, "
+      + "by default the value is 10")
```

Comment:
I don't think you need "by default the value is 10"
jcommander is smart enough to print this http://jcommander.org/#_default_values